### PR TITLE
RFE-3007: Expose option-contstats as an unsupported option

### PR DESCRIPTION
**Pull request by k-wall**
Signed-off-by: kwall <kwall@apache.org>

### CODE REVIEW

**Comment by candita** [comment]
RFE-3007 is still on the backlog.

File: pkg/operator/controller/ingress/deployment.go [hunk 1]
```diff
@@ -80,6 +80,8 @@ const (
 
 	RouterHAProxyConfigManager = "ROUTER_HAPROXY_CONFIG_MANAGER"
 
+	RouterHAProxyContstats = "ROUTER_HAPROXY_CONTSTATS"
```
Comment (Miciah) [inline pkg/operator/controller/ingress/deployment.go:83]: https://github.com/openshift/router/pull/449 adds `ROUTER_CONTSTATS`, and here you have `ROUTER_HAPROXY_CONTSTATS`.

Comment (k-wall) [reply pkg/operator/controller/ingress/deployment.go:83]: @Miciah yes, thank you for spotting. I'd failed to push a change to my upstream branch for https://github.com/openshift/router/pull/449.  It is done now.

**Comment by Miciah** [comment]
/assign

Thanks!  Changes look good.
/approve
/lgtm

Since this PR is adding an option that is explicitly unsupported, it does not need docs, PX, or QE.
/label docs-approved
/label px-approved
/label qe-approved

**Comment by Miciah** [comment]
[e2e-aws-operator failed](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-ingress-operator/887/pull-ci-openshift-cluster-ingress-operator-master-e2e-aws-operator/1681754006537375744) because must-gather failed.  
/test e2e-aws-operator

**Comment by Miciah** [comment]
[e2e-aws-ovn failed](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-ingress-operator/887/pull-ci-openshift-cluster-ingress-operator-master-e2e-aws-ovn/1682165686891188224) because `[sig-instrumentation] Prometheus [apigroup:image.openshift.io] when installed on the cluster should report telemetry` failed.  This test failure is being tracked by [OCPBUGS-16615](https://issues.redhat.com/browse/OCPBUGS-16615).
/test e2e-aws-ovn

[e2e-gcp-operator failed](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-ingress-operator/887/pull-ci-openshift-cluster-ingress-operator-master-e2e-gcp-operator/1682165686878605312) because `TestUnmanagedDNSToManagedDNSInternalIngressController` failed.  This is probably the same issue as [OCPBUGS-2493](https://issues.redhat.com/browse/OCPBUGS-2493).
/test e2e-gcp-operator

**Comment by Miciah** [comment]
/hold cancel
/test all
since #886 merged.

**Comment by Miciah** [comment]
[e2e-aws-operator failed](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-ingress-operator/887/pull-ci-openshift-cluster-ingress-operator-master-e2e-aws-operator/1684066097378103296) because must-gather failed.
/test e2e-aws-operator

**Pull request merged**